src/lib/define-ruby-blocks.js

```diff
@@ -941,6 +941,31 @@ export default function (ScratchBlocks) {
         }
     };
 
+    ScratchBlocks.Blocks.mrubyc_rboard_grove_buzzer_ling = {
+        init: function () {
+            this.jsonInit({
+                type: 'mrubyc_rboard_grove_buzzer_ling',
+                message0: '%1の音のブザーを鳴らす',
+                args0: [
+                    {
+                        type: 'field_dropdown',
+                        name: 'BUZZER',
+                        options: [
+                            ['ド', '0x4F3'],
+                            ['レ', '0x58C'],
+                            ['ミ', '0x639'],
+                            ['ファ', '0x6EF'],
+                            ['ソ', '0x76A'],
+                            ['ラ', '0x84D'],
+                            ['シ', '0x95A']
```

Comment:
PWM.cycleは、
`PWM.cycle(0x95A,4)`
のように第一引数にシングルクォーテーションが無いので、以下のように直してもらえると嬉しいです。

```suggestion
                            ['ド', 0x4F3],
                            ['レ', 0x58C],
                            ['ミ', 0x639],
                            ['ファ', 0x6EF],
                            ['ソ', 0x76A],
                            ['ラ', 0x84D],
                            ['シ', 0x95A]
```

Reply:
承知しました。今すぐ付けます

Reply:
間違えました。今すぐ消します(笑)